src/main/java/controller/RacingGameController.java

```diff
@@ -0,0 +1,55 @@
+package controller;
+
+import racing.Car;
+import racing.RacingGame;
+import view.RacingGameView;
+
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class RacingGameController {
+
+    private RacingGame racingGame;
+    private int tryNums;
+
+    public  void startGame() {
+        setup();
+        playround(racingGame, tryNums);
+        result(racingGame);
+    }
+
+    private void setup() {
+        String carNames = RacingGameView.getCarNames();
+        while (carNames.isEmpty()) {
+            carNames = RacingGameView.getCarNames();
+        }
+        while (true){
+            if (getTryNums()) break;
+        }
+        racingGame = new RacingGame(carNames);
+    }
+
+    private boolean getTryNums() {
+        try {
+            tryNums = RacingGameView.getTryNums();
+            return true;
+        } catch (NumberFormatException e) {
+            System.out.println("숫자를 입력해주세요.");
+        }
+        return false;
+    }
```

Comment:
`함수(또는 메소드)가 한 가지 일만 잘 하도록 구현한다.`의 미션 요구사항에 따라서 `getTryNums`는 TryNums만 입력받는 기능으로 변경해주세요. 

메서드가 여러 가지 일을 한번에 수행하게되면 메서드의 행위에대한 결과를 예측하기 힘들어지고 자연스럽게 유지보수하기가 어려운 코드가 되기 쉽습니다. 

지금은 tryNums를 입력받았을 뿐만 아니라 입력을 정상적으로 받았는지 못받았는지 여부를 알려주는 역할도하고있는 것 같아요. 🤔
아래와 같은 방식으로 변경하게된다면 이 메서드가 하나의 일만 하도록 변경할 수 있을 것 같아요. 😃

```suggestion
    private int getTryNums() {
        try {
            return RacingGameView.getTryNums();
        } catch (NumberFormatException e) {
            System.out.println("숫자를 입력해주세요.");
            return getTryNums();
        }
    }
```